pkgs/development/python-modules/torchao/default.nix

```diff
@@ -176,8 +176,7 @@ buildPythonPackage rec {
     "test_precompute_bitnet_scale"
     "test_qlora_fsdp2"
     "test_uneven_shard"
-  ]
-  ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) [
```

Comment:
Your commit message suggests that this should be `stdenv.hostPlatform.isDarwin`. Are those tests also failing on linux?

Reply:
The start of the block (line 171 above) is `  ++ lib.optionals (stdenv.hostPlatform.isDarwin) [`

The line I'm removing begins tests that appeared to be an aarch64 issue but turned out to break on x86_64-darwin as well.

In other words, this is correct.

Reply:
Oh right. Sorry, my bad.